workflow-ui/compose/src/main/java/com/squareup/workflow1/ui/compose/CompositionRoot.kt

```diff
@@ -84,23 +83,19 @@ public fun ViewRegistry.withCompositionRoot(root: CompositionRoot): ViewRegistry
 
 /**
  * Applies [transform] to each [ViewFactory] in this registry. Transformations are applied lazily,
- * at the time of lookup via [ViewRegistry.getFactoryFor].
+ * at the time of lookup via [ViewRegistry.getEntryFor].
  */
 @WorkflowUiExperimentalApi
 private fun ViewRegistry.mapFactories(
   transform: (ViewFactory<*>) -> ViewFactory<*>
 ): ViewRegistry = object : ViewRegistry {
   override val keys: Set<KClass<*>> get() = this@mapFactories.keys
 
-  override fun <RenderingT : Any> getFactoryFor(
+  override fun <RenderingT : Any> getEntryFor(
     renderingType: KClass<out RenderingT>
-  ): ViewFactory<RenderingT> {
-    val factoryFor =
-      this@mapFactories.getFactoryFor(renderingType) ?: throw IllegalArgumentException(
```

Comment:
This could never have worked, as this method is not and was not the method that swizzles in AndroidViewRendering and such.
    
Compose sample choked on AsIs with this, and spookily works just fine when I delete the check. So...is this transformation actually necessary? Has AndroidViewRendering been broken all along? Will have to check.

Reply:
Tracked in #579